inst/scripts/query-pubmed-grants.R

```diff
@@ -1,26 +1,27 @@
 ## ----setup, include = FALSE-----------------------------------------------------------------------------------------------------------------------------------------------------------
-install.packages('librarian')
-install.packages("synapser", repos=c("http://ran.synapse.org", "http://cran.fhcrc.org"))
+# options(repos = c(CRAN = "https://cloud.r-project.org/"))
```

Comment:
It is best practice to actually include "installation instructions" outside of the script itself.  The script could have something like

`require(librarian)`

which would throw an error if library isn't installed, that is true for all other dependencies as well.